pkg/build/api/types.go

```diff
@@ -707,6 +739,12 @@ type BuildRequest struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// SkipBuildsInQueue will cause all builds in the queue marked as skipped and
+	// make the build created from this build requests be the only build in the
+	// queue. This option is valid only for build configurations with "recent"
+	// build scheduling policy.
+	SkipBuildsInQueue *bool
```

Comment:
not sure this is the way we want to expose that behavior anyway.  i think the use case is more like:

1) i started a build
2) i look at the build queue and see it's full of garbage
3) i run "oc skip-builds" (name tbd) which skips all the queued builds except the most recent one (mine).